src/msg.c

```diff
@@ -687,6 +658,46 @@ static int8_t msg_schema(struct session *session,
 	return result;
 }
 
+static ssize_t send_schema_end_rsp(const struct node_ops *node_ops, int node_fd)
+{
+	knot_msg msg;
+
+	memset(&msg, 0, sizeof(msg));
+
+	msg.hdr.type = KNOT_MSG_SCHM_END_RSP;
+
+	return node_ops->send(node_fd, &msg, sizeof(msg));
+}
+
+static bool handle_schema_updated(struct session *session,
+				  const char *device_id, const char *err)
```

Comment:
remove the unused parameter `device_id`

Reply:
This parameter is been used to get the knot device.